packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -108,6 +120,21 @@ function isSafeUse(node: TSESTree.Node): boolean {
       return true;
 
     case AST_NODE_TYPES.CallExpression:
+      if (
+        parent.callee.type === AST_NODE_TYPES.MemberExpression &&
+        parent.callee.property.type === AST_NODE_TYPES.Identifier
+      ) {
+        const method = parent.callee.property.name;
+        const args = parent.arguments;
+
+        if (Object.values(MethodsWithThisArg).includes(method)) {
```

Comment:
if there is other way to check this, let me know

Reply:
```suggestion
        if (methodsWithThisArg.has(method)) {
```